### ISSUE TITLE
xia2 pipeline=dials processing Eiger demo data didn't

### ISSUE THREAD

**Issue by graeme-winter** [xia2/xia2#90]
```
------------------- Unit cell refinement -------------------
SAD:  77.50  77.50  77.50  90.00  90.00  90.00
Status: error "1483525015.491323"
Please send the contents of xia2.txt, xia2.error and xia2-debug.txt to:
xia2.support@gmail.com
```

Error was the following:

```
[gw56@cs04r-sc-com99-01 001-dials]$ cat xia2.error
Traceback (most recent call last):
  File "/dls/science/users/gw56/svn/cctbx/build/../modules/xia2/command_line/xia                                                                                                                                                                                               2_main.py", line 282, in run
    xinfo = xia2_main()
  File "/dls/science/users/gw56/svn/cctbx/build/../modules/xia2/command_line/xia                                                                                                                                                                                               2_main.py", line 233, in xia2_main
    Chatter.write(xinfo.get_output(), strip=False)
  File "/dls/science/users/gw56/svn/cctbx/modules/xia2/Schema/XProject.py", line                                                                                                                                                                                                130, in get_output
    result += self._crystals[crystal].get_output()
  File "/dls/science/users/gw56/svn/cctbx/modules/xia2/Schema/XCrystal.py", line                                                                                                                                                                                                474, in get_output
    reflections_all = self.get_scaled_merged_reflections()
  File "/dls/science/users/gw56/svn/cctbx/modules/xia2/Schema/XCrystal.py", line                                                                                                                                                                                                867, in get_scaled_merged_reflections
    return self._get_scaler().get_scaled_merged_reflections()
  File "/dls/science/users/gw56/svn/cctbx/modules/xia2/Schema/Interfaces/Scaler.                                                                                                                                                                                               py", line 662, in get_scaled_merged_reflections
    self.scale()
  File "/dls/science/users/gw56/svn/cctbx/modules/xia2/Schema/Interfaces/Scaler.                                                                                                                                                                                               py", line 629, in scale
    self._scale_finish()
  File "/dls/science/users/gw56/svn/cctbx/modules/xia2/Modules/Scaler/CommonScal                                                                                                                                                                                               er.py", line 548, in _scale_finish
    self._scale_finish_chunk_2_report()
  File "/dls/science/users/gw56/svn/cctbx/modules/xia2/Modules/Scaler/CommonScal                                                                                                                                                                                               er.py", line 601, in _scale_finish_chunk_2_report
    batch_to_dose = self.get_batch_to_dose()
  File "/dls/science/users/gw56/svn/cctbx/modules/xia2/Modules/Scaler/CCP4Scaler                                                                                                                                                                                               A.py", line 1397, in get_batch_to_dose
    batch_to_dose[b] = epoch_to_dose[si._image_to_epoch[b-batch_offset]]
KeyError: 1483525015.491323
```

Worked in :/dls/tmp/gw56/eiger-problem/001-dials

Command was the unremarkable "xia2 atom=S pipeline=dials ../2015_11_10/"

Logging here so does not get lost...

**Comment by graeme-winter** [comment]
Identical error for pipeline=3di

**Comment by graeme-winter** [comment]
Working on this now...

**Comment by graeme-winter** [comment]
Looks like this works just fine now; tested multiple examples

**Comment by Anthchirp** [comment]
@dagewa's run:

[xia2.txt](https://github.com/xia2/xia2/files/713844/xia2.txt)
[xia2.error.txt](https://github.com/xia2/xia2/files/713847/xia2.error.txt)
[xia2-debug.txt](https://github.com/xia2/xia2/files/713845/xia2-debug.txt)

**Comment by Anthchirp** [comment]
fixed by 814dfbe35efc26b6cd9384bcafe4fa4bd68beef1
